django/db/models/sql/compiler.py

```diff
@@ -150,7 +147,14 @@ def collapse_group_by(self, expressions, having):
                         getattr(expr.output_field, 'model') == self.query.model):
                     pk = expr
             if pk:
+                # MySQLism: Columns in HAVING clause must be added to the GROUP BY.
```

Comment:
I couldn't agree more but this is no invention of mine. I simply moved this comment down from the line 140.